### PR TITLE
Improve style, add libre game filter

### PR DESCRIPTION
**Pull request by alexgleason**
Hi again,

This change is untested because I could not figure out how. Please test it before merging.

I improved the CSS on the homepage to look like this:

screenshot from 2018-02-19 14 05 02

I also included a libre game filter that only displays games with certain licenses.

Finally, if a game doesn't have a license, that makes the game proprietary, so I changed the language from "unspecified" to "proprietary" and removed the icon in that case. Hopefully Mustache works the way I expect here.

### CODE REVIEW

**Comment by eried** [comment]
Awesome! I'll test it soon and publish it

**Review by eried — CHANGES_REQUESTED** [review]
There is an error with the mustache syntax. You can check the reference here: https://mustache.github.io/mustache.5.html and you can paste the `template.html` here https://mustache.github.io/#demo in the first field and http://arduboy.ried.cl/repo.json contents in the second field to test it without installing anything.

File: .admin/public_html/arduboy/template.html [hunk 1]
```diff
@@ -47,12 +83,16 @@ <h1>{{repository}}</h1>
 					<!-- Boxes -->
 						<div class="thumbnails">
 {{#items}}
-							<div class="box" data-item-tags="{{genre}}" data-item-date="{{date}}">
+							<div class="box" data-item-tags="{{genre}}" data-item-date="{{date}}" data-item-license="{{license}}">
 								<a class="image fit"><img src="{{banner}}" alt="" /></a>
 								<div class="inner">
 									<h3>{{title}} by <a href="{{sourceUrl}}" title="View source code" target="_blank" >{{author}}</a></h3>
 									<p>{{description}}</p>
+									{{#license}}
```
Comment (eried) [inline .admin/public_html/arduboy/template.html:91]: [20-Feb-2018 17:04:21 America/Chicago] PHP Fatal error:  Uncaught exception 'Mustache_Exception_SyntaxException' with message 'Nesting error: license (on line 90) vs. items (on line 103)' in /home/erwinrie/public_html/arduboytest/Mustache/Parser.php:126
Stack trace:
#0 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(105): Mustache_Parser->buildTree(Array, Array)
#1 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(105): Mustache_Parser->buildTree(Array, Array)
#2 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(43): Mustache_Parser->buildTree(Array)
#3 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(745): Mustache_Parser->parse(Array)
#4 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(759): Mustache_Engine->parse('<!DOCTYPE HTML>...')
#5 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(700): Mustache_Engine->compile('<!DOCTYPE HTML>...')
#6 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(619): Mustache_Engine->loadSource('<!DOCTYPE HTML>...')
#7 /home/erwinrie/public_html/arduboytest/Mustache/ in /home/erwinrie/public_html/arduboytest/Mustache/Parser.php on line 126

**Comment by alexgleason** [comment]
Okay, yes! My mustache syntax was wrong. I just force-pushed the update to this branch and tested it locally; it works.

**Review by eried — APPROVED** [review]

**Review by eried — APPROVED** [review]

**Comment by eried** [comment]
Changes were included in https://github.com/eried/ArduboyCollection/pull/36

**Pull request closed without merging**